s3/src/main/scala/geotrellis/spark/io/s3/S3LayerUpdater.scala

```diff
@@ -37,71 +38,40 @@ class S3LayerUpdater(
 ) extends LayerUpdater[LayerId] with LazyLogging {
 
   def rddWriter: S3RDDWriter = S3RDDWriter
+  def _rddWriter(): S3RDDWriter = rddWriter
+
+  class InnerS3LayerWriter(
```

Comment:
Mb to make this class private? Or add some overloads to make construction of `S3LayerWriter` with a custom `rddWriter` more convenient?